plugins/rapid7_insightidr/help.md

```diff
@@ -40,6 +47,411 @@ Example input:
 
 ### Actions
 
+#### List Alerts for Investigation
+
+This action is used to retrieve a page of alerts associated with the specified investigation.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|string|None|True|The ID or RRN of the investigation|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|index|integer|0|True|The optional zero-based index of the page to retrieve. Must be an integer greater than or equal to 0|None|1|
+|size|integer|100|True|The optional size of the page to retrieve. Must be an integer greater than 0 or less than or equal to 100. Default value is 100|None|100|
+
+Example input:
+
+```
+{
+  "id": "rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111",
+  "index": 1,
+  "size": 100
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|alerts|[]alert|True|A list of alerts associated with the investigation|
+|metadata|investigation_metadata|True|The pagination parameters used to generate this page result|
+
+Example output:
+
+```
+{
+  "alerts": [
+    {
+      "alert_type": "Example Type",
+      "alert_type_description": "Example Description",
+      "created_time": "01-01-2020T00:00:00",
+      "detection_rule_rrn": {
+        "rule_name": "Example Rule Name",
+        "rule_rrn": {
+          "organizationId": "11111111-1111-1111-1111-111111111111",
+          "regionCode": "11-101",
+          "resource": "Example Resource",
+          "resourceTypes": [
+            "Example Type"
+          ],
+          "service": "Example Service"
+        }
+      },
+      "first_event_time": "01-01-2020T00:00:00",
+      "id": "11111111-1111-1111-1111-111111111111",
+      "latest_event_time": "01-01-2020T00:00:00",
+      "title": "Example Title"
+    }
+  ],
+  "metadata": {
+    "index": 0,
+    "size": 1,
+    "total_data": 1,
+    "total_pages": 1
+  }
+}
+```
+
+#### Set Priority of Investigation
+
+This action is used to change the priority of the investigation with the given ID or RRN.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|string|None|True|The ID or RNN of the investigation to change the priority of|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|priority|string|None|True|Investigation's priority|['UNSPECIFIED', 'LOW', 'MEDIUM', 'HIGH', 'CRITICAL']|LOW|
```

Comment:
In Update priority None is an accepted value. Can you please check if it is an accepted value for set priority

Reply:
Empty is not accepted value for `Set priority of investigation` action